docs/source/support/faq.rst

```diff
@@ -140,7 +140,7 @@ Inflation Rewards (NU) and Policy Fees (ETH).
 Q: How are Policy Fees (ETH) determined?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The fee is calculated based on each period committed to and the number of policies that the Ursula is enforcing.
+The amount of fees earnable by a staker in a given period is determined by multiple factors, including (1) the minimum fee rate set by the staker, (2) the number of active policies issued by users of the network to that staker, and (3) whether the staker has previously committed to being available for that period. See `Service Fees (Pricing) <https://docs.nucypher.com/en/latest/architecture/service_fees.html>`_ for details.
```

Comment:
@arjunhassard I'm never one to question your vocabulary prowess 😅 - but is "earnable" a word?

Reply:
"Earnable" sounds good to me, and seems to have at least some marginal use for those that speak legalese, but I guess we can phrase it in a simpler way:
```suggestion
The amount of fees a staker can earn in a given period is determined by multiple factors, including (1) the minimum fee rate set by the staker, (2) the number of active policies issued by users of the network to that staker, and (3) whether the staker has previously committed to being available for that period. See `Service Fees (Pricing) <https://docs.nucypher.com/en/latest/architecture/service_fees.html>`_ for details.
```

Reply:
The FAQ has turned into a nursery for actual documentation.

Reply:
@KPrasch Isn't that kind of what an FAQ is for? To give users a quick, simplified answer without having to trudge through the whole spec – thereby addressing high-level confusion before a rage/exhaustion-quit? 

Do you think this change makes it more or less of a nursery? I corrected a misleading answer and added a link to the actual documentation.

Reply:
I'm not saying it's a bad thing, just an observation.  Even before this PR, the FAQ encompasses too much essential information to be properly composed and discovered here.  I even think it's part of the lifecycle of documentation for our organization, certainly better to have *some* kind of reference in the form of an FAQ than none at all.

Reply:
@KPrasch agreed, it's particularly intimidating to a fresh developer/user to have to navigate through so much information primarily aimed at stakers. been saying for a long time we need to split all our educational materials by audience (demand-side and service-side) – maybe now's the time?

Reply:
Our old friend #1001

Reply:
Some initial work by @KPrasch , #2463 .